Gems/EMotionFX/Code/EMotionFX/Source/ActorManager.cpp

```diff
@@ -31,7 +31,7 @@ namespace EMotionFX
         SetScheduler(MultiThreadScheduler::Create());
 
         // reserve memory
-        m_actors.reserve(512);
+        m_actorAssets.reserve(32);
         m_actorInstances.reserve(1024);
         m_rootActorInstances.reserve(1024);
```

Comment:
We can get rid of these entirely, seems pretty random and the growing of the AZStd::vector will take care of growing/allocating some extra space always.